src/ext/config.js

```diff
@@ -9,35 +9,44 @@
 /* exported ACTIVE_CONFIG */
 /* exported PPConfigs */
 
-const CHL_BYPASS_SUPPORT  = "cf-chl-bypass"; // header from server to indicate that Privacy Pass is supported
+const CHL_BYPASS_SUPPORT = "cf-chl-bypass"; // header from server to indicate that Privacy Pass is supported
 const CHL_BYPASS_RESPONSE = "cf-chl-bypass-resp"; // response header from server, e.g. with erorr code
 
+function cloneConfig(config) {
+    return JSON.parse(JSON.stringify(config));
+}
+
 const exampleConfig = {
     "id": 0, // unique integer identifying each individual config
     "dev": true, // sets whether the configuration should only be used in development
     "sign": true, // sets whether tokens should be sent for signing
     "redeem": true, // sets whether tokens should be sent for redemption
-    "sign-reload": true, // whether pages should be reloaded after signing tokens (e.g. to immediately redeem a token)
-    "sign-resp-format": "string", // formatting of response to sign request (string or json)
-    "max-spends": 3, // for each host header, sets the max number of tokens that will be spent
+    "max-spends": 3, // for each host header, sets the max number of tokens that will be spent, 0 for unlimited
```

Comment:
Nit: can we change this comment to say `undefined` for unlimited.

Reply:
Great catch, fixed!